.github/workflows/linux_arm64.yml

```diff
@@ -116,8 +116,26 @@ jobs:
       - name: Install build dependencies
         run: |
           bash ${OPENVINO_REPO}/install_build_dependencies.sh
-          # default-jdk - Java API
-          apt install --assume-yes --no-install-recommends default-jdk
+          
+          # default-jdk - Java API, software-properties-common for add-apt-repository
+          apt install --assume-yes --no-install-recommends default-jdk software-properties-common
+          
+          # Install gcc-10
+          add-apt-repository ppa:ubuntu-toolchain-r/test
```

Comment:
do we need it? looks like in order to use this repo, we have to call apt-get update,, but it's not done :)